### DISCUSSION TITLE
How many features can qfield display? Problem with large amount of features in qfield

### DISCUSSION THREAD

**Discussion by Dachs20** [opengisch/QField#5167] [Q&A]
I have created a project in Qgis for Qfield and loaded it into the qfieldcloud. It contains a geopackage that now contains about 1370 features. until recently everything ran smoothly.  Then the geopackage was no longer displayed with the features in qfield. 
the data is available on the cloud and in Qgis works without problems. Then I deleted about 60 Fearures. After that, the geopackage with the features was displayed again in Qfield. The Fearutes that were deleted were already older and were displayed without any problems.
Is there an upper limit of features that Qfield can display?

**Comment by BoswachterMarc** [comment]
I don't think there is a limit. Our project runs smoothly with about 30.000 spatial features and 10.000 non-spatial records in different layers in one geopackage.

**Comment by Oisin-JK** [comment]
I also regularly used QField Projects with thousands of features  with no problem on a Samsung S22.
